gensim/test/test_BM25.py

```diff
@@ -62,6 +62,76 @@ def test_multiprocessing(self):
         self.assertAlmostEqual(weights1, weights3)
         self.assertAlmostEqual(weights2, weights3)
 
+    def test_k1(self):
+        """ Changing the k1 parameter should give consistent results """
+        corpus = common_texts
+        index = 0
+        doc = corpus[index]
+        first_k1 = 1.0
+        second_k1 = 2.0
+
+        first_bm25 = BM25(corpus, k1=first_k1)
+        second_bm25 = BM25(corpus, k1=second_k1)
+        self.assertTrue(first_bm25.get_score(doc, index) < second_bm25.get_score(doc, index))
```

Comment:
Simplify the assertion:

```suggestion
        first = first_bm25.get_score(doc, index)
        second = second_bm25.get_score(doc, index)
        self.assertLessThan(first, second)
```

This will give a more helpful message upon failure.

Reply:
I simplified the assertions in 4a61276.